truss/templates/trtllm/packages/schema.py

```diff
@@ -40,6 +40,70 @@ def __init__(
         self._top_p = top_p
         self._top_k = top_k
 
+    @staticmethod
+    def from_bridge_oai_request(
+        model_input,
+        chat_templater: Callable[[Any], Any],
+        request_id: str,
+        eos_token_id: str,
+    ) -> "ModelInput":
+        """
+        The BridgeCompletionRequest is input into the `from_bridge_oai_request` function.
+        This model is json-serialized as input into the `predict` endpoint.
+        If changes are needed, reach out the Baseten Core Product team
+
+        class BridgeCompletionRequest(BaseModel):
```

Comment:
Should we just copy this over for right now and "update both" any time we update this definition?